### PR TITLE
Fix Personal Sign Display

### PR DESCRIPTION
**Pull request by samgbafa**
## Explanation
This PR makes a change to the personal_sign page. Currently there is an icon that is rendered based on the address passed, but the address passed is `null`. This PR changes the address passed to use the same value that is used to select the account (`fromAccount`).

## More Information
Related to https://github.com/MetaMask/metamask-extension/pull/14438#issuecomment-1204225916

### Before

Before

### After
After

## Manual Testing Steps
Go to the [test-dapp](https://metamask.github.io/test-dapp/) and hit personal sign

## Pre-Merge Checklist

- [ ] PR template is filled out
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [ ] PR is linked to the appropriate GitHub issue
- [ ] PR has been added to the appropriate release Milestone

### + If there are functional changes:

- [ ] Manual testing complete & passed
- [ ] "Extension QA Board" label has been applied

### CODE REVIEW

File: ui/components/app/signature-request-original/signature-request-original.component.js [hunk 1]
```diff
@@ -228,7 +229,8 @@ export default class SignatureRequestOriginal extends Component {
               className="request-signature__help-link"
               onClick={() => {
                 global.platform.openTab({
-                  url: 'https://consensys.net/blog/metamask/the-seal-of-approval-know-what-youre-consenting-to-with-permissions-and-approvals-in-metamask/',
+                  url:
```
Comment (skgbafa) [inline ui/components/app/signature-request-original/signature-request-original.component.js:232]: My linter auto-changed this, I can revert if needed

Comment (georgewrmarshall) [reply ui/components/app/signature-request-original/signature-request-original.component.js:232]: Hmm yup linter has an issue haha. Try `yarn lint:fix` should fix it

Comment (skgbafa) [reply ui/components/app/signature-request-original/signature-request-original.component.js]: Done!

**Review by georgewrmarshall — COMMENTED** [review]
I just want to confirm what the intention of the Identicon is for. We are already showing the users account avatar that is generated from their address. Showing it again doesn't make a huge amount of sense? The original address variables is called  `requesterAddress` so that makes me think that the intention is for it to be an identifier avatar for the contract address. If we can't get that should we just remove? @rachelcope @SayaGT

**Comment by SayaGT** [comment]
@georgewrmarshall Not sure what the identicon is supposed to show (it has always showed up blank for me) but with the context it should show the icon/ logo for the dApp requesting the signature. Doesn't make sense to show the account avatar again. We can remove it as long as the icon is displayed in the origin along with the dApp URL. 

Also, this soon will be replaced by new designs @holantonela has worked on which solve these issues.

**Comment by NiranjanaBinoy** [comment]
We are having `requesterAddress` in three components set from respective containers to render the icon using <Identicon/> , SignatureRequestOriginal ConfirmEncryptionPublicKey, and  ConfirmDecryptMessage, but in all three places `requesterAddress` is assigned to `null` in  the respective containers. Based on the above comment by @SayaGT maybe we can remove it from all three places? 

cc @danjm and @Gudahtt to know if they have more context for the use of  `requesterAddress`.

**Comment by Gudahtt** [comment]
`requesterAddress` has always been `null`. See [the original PR here](https://github.com/MetaMask/metamask-extension/pull/2479). At one point it showed the Ethereum logo as a default, that must have broken at some point.

I agree that it doesn't make sense to show the account avatar twice. Leaving this as-is or removing it would make the most sense to me for now. The new designs will certainly solve this for us.

**Comment by georgewrmarshall** [comment]
+1 vote for removing

**Review by brad-decker — APPROVED** [review]

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Just blocking this PR I think we can fully remove the `renderRequestIcon` from rendering the Identicon here

**Comment by brad-decker** [comment]
@skgbafa could you remove the renderRequestIcon as per @georgewrmarshall 's request so we can move this forward

**Comment by bschorchit** [comment]
I believe we're removing this icon already in this PR: https://github.com/MetaMask/metamask-extension/pull/15776

**Review by brad-decker — APPROVED** [review]

**Comment by brad-decker** [comment]
@bschorchit @georgewrmarshall is this PR still needed then? Does this mean its good to go as is?

**Comment by bschorchit** [comment]
> @bschorchit @georgewrmarshall is this PR still needed then? Does this mean its good to go as is? 

I don't think this PR is needed. Let's prioritize https://github.com/MetaMask/metamask-extension/pull/15776 and the issue that motivated this PR won't exist anymore 🙂

**Comment by brad-decker** [comment]
Closing in favor of #15776

**Pull request closed without merging**